lineage/figures/figureCommon.py

```diff
@@ -148,7 +148,7 @@ def overlayCartoon(figFile, cartoonFile, x, y, scalee=1, scale_x=1, scale_y=1):
     template.save(figFile)
 
 
-def figureMaker(ax, x, paramEst, accuracies, tr, pii, paramTrues, xlabel="Number of Cells"):
+def figureMaker(ax, x, paramEst, accuracies, tr, pii, paramTrues, balanced_acc, xlabel="Number of Cells"):
```

Comment:
`balanced_acc` can be removed from this function list of arguments, or change `accuracies` to `balanced_accuracies` and remove `balanced_acc`. Make sure that in the rest of the function `balanced_accuracies` is replacing `accuracies`.

Reply:
Pretty sure the figures won't build unless this change is made.

Reply:
See here:
```
Traceback (most recent call last):
  File "./genFigures.py", line 20, in <module>
    ff = eval(nameOut + '.makeFigure()')
  File "<string>", line 1, in <module>
  File "/home/jenkins/workspace/meyer-lab_lineage-growth_fig4/lineage/figures/figureS02.py", line 34, in makeFigure
    figureMaker(ax, *accuracy())
TypeError: figureMaker() missing 1 required positional argument: 'balanced_acc'
```

Reply:
The figures should be building before this is merged.